doc-distrib-ws/src/main/resources/application.yaml

```diff
@@ -11,6 +17,14 @@ mailservice:
 
 ws:  
   faxFormat: IMCEAFAX-%RECIPIENT%+40%FAXNUMBER%@Victoria1.gov.bc.ca
+  
+  # Timeout (in milliseconds) when no reponse has been received by exhange for a sent fax
+  #   After this timeout, the fax is considered lost and a failed status is returned to the user.  
+  faxTimeout: 1500000    # 25 minutes 
```

Comment:
Adding a comment with a conversion to a more readable time like minutes or seconds is a good idea, I like this.